classes/qa-denydeps.bbclass

```diff
@@ -26,21 +26,24 @@ python package_qa_check_denydeps() {
             path = [package]
         if package in denylist:
             bb.error(package + " in RDENYDEPENDS but found via " + repr(path))
-            return
+            return False
         if package in visited:
-            return
+            return True
 
         visited[package] = True
 
         pkgdata = oe.packagedata.read_subpkgdata(package, d)
 
         if ('RDEPENDS_' + package) not in pkgdata:
-            return
+            return True
 
         rdepends = bb.utils.explode_deps(pkgdata['RDEPENDS_' + package] or "")
 
         for rdepend in rdepends:
-            check_deps(rdepend, denylist, visited, path + [rdepend])
+            if not check_deps(rdepend, denylist, visited, path + [rdepend]):
+                return False
+
+        return True
```

Comment:
I think the second commit has some fixup diffs for `qa-denydeps.bbclass` that should have been squashed into the previous commit.

Reply:
ugh, oops. fixed now.